src/com/palmergames/bukkit/towny/command/PlotCommand.java

```diff
@@ -688,25 +770,29 @@ public static void setTownBlockPermissions(Player player, TownBlockOwner townBlo
 					perm.set(s, b);
 				} catch (Exception e) {
 					TownyMessaging.sendErrorMsg(player, TownySettings.getLangString("msg_plot_set_perm_syntax_error"));
-					return;
+					return false;
 				}
 
 			}
 
 			townBlock.setChanged(true);
 			townyUniverse.getDataSource().saveTownBlock(townBlock);
-
-			TownyMessaging.sendMsg(player, TownySettings.getLangString("msg_set_perms"));
-			TownyMessaging.sendMessage(player, (Colors.Green + " Perm: " + ((townBlockOwner instanceof Resident) ? perm.getColourString().replace("n", "t") : perm.getColourString().replace("f", "r"))));
-			TownyMessaging.sendMessage(player, (Colors.Green + " Perm: " + ((townBlockOwner instanceof Resident) ? perm.getColourString2().replace("n", "t") : perm.getColourString2().replace("f", "r"))));
-			TownyMessaging.sendMessage(player, Colors.Green + "PvP: " + ((perm.pvp) ? Colors.Red + "ON" : Colors.LightGreen + "OFF") + Colors.Green + "  Explosions: " + ((perm.explosion) ? Colors.Red + "ON" : Colors.LightGreen + "OFF") + Colors.Green + "  Firespread: " + ((perm.fire) ? Colors.Red + "ON" : Colors.LightGreen + "OFF") + Colors.Green + "  Mob Spawns: " + ((perm.mobs) ? Colors.Red + "ON" : Colors.LightGreen + "OFF"));
+			
+			if (!townBlock.hasPlotObjectGroup()) {

```

Comment:
ohh ok i see what the problem you had with this was originally, the if statement needs to be there otherwise the console will be spammed with all of the plot perm changes for every town block even though they're the same. I added code in the confirmation handler to provide output only once.

Reply:
I saw that, but in the case that someone is changing just one of the plots in the group using ```/plot set perm something something``` because it's in a group they wont see their changes as an output.

Reply:
Here's the deal I'm gonna leave it so perms are set throughout the group and for now individual plot perms can't bet set if people ask for the feature, then it will be added later. Right now I'm going to tell the user to use /plot group set perm if the block is in a group.

Reply:
Sounds good. I'd always pictured it where a group has the same perm line throughout anyways.